cvat-core/src/lambda-manager.ts

```diff
@@ -43,7 +43,7 @@ class LambdaManager {
         return { models, count: lambdaFunctions.length };
     }
 
-    async run(taskID: number, model: MLModel, args: any) {
```

Comment:
Promise<any> should be specified more concrete in this file

Reply:
@tahamukhtar20

Reply:
I'll get to these tomorrow after submitting my proposal. Thanks for the reminder.